File: 2023/01 - January/06/01/Node.java
// Node.java

// Node Class
public class Node {
	// Private Attributes
	private int data;
	private Node next;

	// Constructors
	public Node(int data) {
		this(data, null);
	}

	public Node(int data, Node next) {
		this.data = data;
		this.next = next;
	}

	// Returns The Data
	public int getData() {
		return data;
	}

	// Returns Next Node
	public Node getNext() {
		return next;
	}

	// Sets Next Node
	public void setNext(Node next) {
		this.next = next;
	}
}

File: 2023/03 - March/13/02/Node.cpp
#include "Node.h"

// Constructor
Node :: Node(Person * data, Node * next) {
	// Initialize Instance Variables
	this->data = data;
	this->next = next;
}

// Setter For Next
void Node :: setNext(Node * next) {
	this->next = next;
}

// Getter For Next
Node * Node :: getNext() {
	return this->next;
}
// Getter For Data
Person * Node :: getData() {
	return this->data;
}

